charts/external-nodes/templates/nodes-cm.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
```

Comment:
rename the filename to node-configmap

Reply:
i think -cm is good enough:
* `kubectl get cm` - so it is formal 
* all our cf-helm charts uses this naming